README.md

```diff
@@ -59,18 +59,13 @@ solutions. If you cannot find your specific problem then please create a
 
 Adding a new component
 ---
-#### Install our [yeoman component generator](https://github.com/holidayextras/generator-ui-component)
-```bash
-npm install -g holidayextras/generator-ui-component
-```
-
 #### Creating the component
 Generate a component & follow the prompts
 ```bash
 cd src/components
 mkdir your-new-component
 cd your-new-component
-yo ui-component:external
+~~yo ui-component:external~~
```

Comment:
Should we just remove this line if we are removing the link to install the generator? There are also references to it in https://github.com/holidayextras/ui-toolkit/blob/master/CONTRIBUTING.md should these also be removed?